### ISSUE TITLE
tutorial not available 

### ISSUE THREAD

**Issue by rogeliodh** [azimux/externals#10]
Hi, it seems that http://nopugs.com/ext-tutorial is down.

Is the tutorial available anywhere else? You could put it in the wiki here.

**Comment by azimux** [comment]
Oh yeah, you're right.  I brought that page back online.

Moving it to the wiki is a great idea and not to hard to do.  I should knock that out.

Note: that article is super old.  The README contains a lot of good info that is helpful.  Running `ext --help` is also helpful.  There are also a couple other externals articles in http://nopugs.com/category/externals

**Comment by rogeliodh** [comment]
yeah, I already figured out how to use it, but the tutorial would have been way faster :)

Thanks for bringing up your server again.

Moving it to the wiki, would be nice. This way, people could contribute and update the info there.

**Comment by azimux** [comment]
totes, I should move it

**Comment by azimux** [comment]
Circling back to this eons later... the URL works again. Thanks for the GitHub issue!
